scripts/__init__.py

```diff
@@ -28,6 +28,7 @@
 
 SUCCESS = 0
 FAILURE = 1
+HAS_ERROR = 2
```

Comment:
I would have `grep` semantics for the return value here: i.e., 0 for success, 1 for errors, and 2 for failures.